patches/06-build-darwin-binary.patch

```diff
@@ -1,13 +1,13 @@
 diff -up ./Makefile.darwin ./Makefile
---- ./Makefile.darwin	2021-02-18 12:02:57.158711869 -0500
-+++ ./Makefile	2021-02-18 12:10:11.759446770 -0500
-@@ -195,3 +195,9 @@ help: ## Show this help screen.
+--- ./Makefile.darwin	2024-07-25 17:30:44.164036603 +0530
++++ ./Makefile	2024-07-25 17:32:13.983235216 +0530
+@@ -208,3 +208,9 @@ help: ## Show this help screen.
  	@echo 'Available targets are:'
  	@echo ''
  	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-25s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 +
 +.PHONY: build-darwin
-+build-darwin: ## Build operator-sdk, ansible-operator, and helm-operator.
```

Comment:
Are we removing `helm-operator` because the command itself is not calling it?

Reply:
The bug (https://bugzilla.redhat.com/show_bug.cgi?id=1930356) was originally reported for only operator-sdk binary. As you can see, the only operator-sdk is being built by the make target. I think that when the patch was created originally (https://github.com/openshift/ocp-release-operator-sdk/pull/104), the comment was simply copied from the build target in the main Makefile, however, the target is only used for building operator-sdk.